DeDRM_plugin/adobekey.py

```diff
@@ -51,12 +53,25 @@
 """
 Retrieve Adobe ADEPT user key.
 """
-from __future__ import print_function
-
 __license__ = 'GPL v3'
 __version__ = '6.0'
 
 import sys, os, struct, getopt
+from base64 import b64decode
+try:
+    import six
+except ImportError:
+    # six may not be available when this script is executed via wine
```

Comment:
`wine python -m pip install six` does not work?!?

Reply:
It would. I wanted to simplify installation for Linux users, and the idea was that if I can do that by vendoring parts of six, it would be worth it. I realize now that Linux users have to install PyCrypto anyway, so it’s not a big effort to install six as a dependency at the same time as well. I’ll change this.